src/components/text-editor/types.ts

```diff
@@ -0,0 +1,16 @@
+import { NodeSpec } from 'prosemirror-model';
+
+/**
+ * @private
+ * set to private to avoid usage while under development
+ */
+export type TextEditorPlugin = {
+    node: NodeConfig[];
```

Comment:
We can't expose types from a third party like this. If there's a breaking change in these types those would be immediately exposed from lime elements without us necessarily being aware of it. We need to create our own type and then map that type to the `NodeSpec` type

Reply:
I think you might have conflated NodeConfig with NodeSpec? :)

Reply:
Yes :slightly_smiling_face: